### PR TITLE
Refactor and structuring update

### PR DESCRIPTION
**Pull request by Dylan-Graham**
Creating PR to leave feedback on

### CODE REVIEW

File: api/app/main/service/sportService.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import os
+from app.main import dbContext
+from app.main.model import windModel
+
+class sportService():
+
+    supportedSports = [
```
Comment (Dylan-Graham) [inline api/app/main/service/sportService.py:7]: Could we make this equal to the sport array in the config.py? Then we are just maintaining the sports in one place

File: api/app/server_old.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# from app.entities import dankOMeter
```
Comment (Dylan-Graham) [inline api/app/server_old.py:1]: Can we delete this? We can always look at previous commits to see the old server

File: api/app/main/service/sportService.py (hunk 1 above)
Comment (Dylan-Graham) [reply api/app/main/service/sportService.py:7]: Or remove the sport array in the config

File: api/app/server_old.py (hunk 2 above)
Comment (Zingers-ZA) [reply api/app/server_old.py:1]: Sweet

File: api/app/main/service/sportService.py (hunk 1 above)
Comment (Zingers-ZA) [reply api/app/main/service/sportService.py:7]: Yeah, I wasn't sure if we should put the supported sports in the db, or leave it in the config. But, I don't see any reason to store it in the db, so I'm happy to leave it stored in config. Will make the change.

**Pull request merged**